doc/whats_new.rst

```diff
@@ -165,10 +172,15 @@ Bug
 
 - Fix error in mne coreg when saving with scaled MRI if fiducials haven't been saved by `Ezequiel Mikulan`_
 
+- Fix normalization error in :func:`mne.beamformer.make_lcmv` when ``pick_ori='normal', weight_norm='unit_noise_gain'`` by `Marijn van Vliet`_
+
+- Fix computation of max-power orientation in :func:`mne.beamformer.make_dics` when ``pick_ori='max-power', weight_norm='unit_noise_gain'`` by `Marijn van Vliet`_
 
 API
 ~~~
 
+- The parameter ``rank`` was added to :func:`mne.beamformer.make_dics` and :func:`mne.beamformer.make_lcmv`` with a default of ``'full'`` that will change to ``None`` in 0.18 to auto-compute the rank of the covariance matrix before regularization by `Marijn van Vliet`_
```

Comment:
@wmvanvliet I realized that we should probably do this. We need a deprecation cycle for `make_lcmv` to do this. I assume we also need it for `make_dics` but if not let me know and I can set the defaults there back to `None` instead of `''` (the internal signal I used for checking deprecation).

Reply:
The default for `make_dics` is already `None`. For the LCMV, I don't like changing the default without some good simulation study backing it up. Both the LCMV and DICS defaults are carefully selected to yield generally stable beamformers.

Reply:
@britta-wstnr care to weight in on this?